checksums.json

```diff
@@ -1,587 +1,12 @@
-        "md5": "0f8dcb17d80368c8af568e218d965287"
-    },
-    "3.5.0.20": {
-        "sha1": "768d4011d22b5c6255d19d4757331d9a34865f77",
-        "sha256": "71597059d50c09d82a0313c3a0d688e00de2a5f6a48bc95e56e34d11d10b1feb",
-        "md5": "3d83972f897d6f99b3e843eb44258495"
-    },
-    "4.4.0.8": {
-        "sha1": "bb07b4e98a0938312583363016e753d1bc108453",
-        "sha256": "c6caea9a70dcb03e6d02c4a1d32655beace39a1d3282df29b171d205543e351e",
-        "md5": "c9ab5c2d96e5300d4d0415b617b40883"
-    },
-    "4.3.0.7": {
-        "sha1": "6f97d9006ff792082a81861caaa97133202d9c3a",
-        "sha256": "f684640871a29e1b3c0877342be5c9af95b229bcd4d9559d9d0327e6a6be0971",
-        "md5": "9c718c44acddcc4f57b0b3cef4d13e56"
-    },
-    "3.5.0.19": {
-        "sha1": "7528329810a25400fab5daf94e4a0b318d8efb57",
-        "sha256": "5165bdd3f4b40faf9b18116b21e244bd7aa6f7442c0bbb4174da84370daf2cd1",
-        "md5": "8bc1a2c9cae2081c3599ed8989ec9064"
-    },
-    "4.4.0.7": {
-        "sha1": "657525b5c854026325ecf739d0a91ac4d4003921",
-        "sha256": "58fcf00c4e50075714cc88a3d345dd718c79400f5beff93c1bbc7387e4a079eb",
-        "md5": "49a52964f5b22d9e94df114e141626b4"
-    },
-    "3.5.0.18": {
-        "sha1": "480997395897718bab8a89262952f7971d098ca2",
-        "sha256": "d3e42a902f7e09ffe74adf036038bd654b67c5698a1c333a3870480f9018c250",
-        "md5": "d3fde0c798521dc70a57b7634f217b87"
-    },
-    "4.3.0.6": {
-        "sha1": "3702ff862d7a546974346c021d647180f47d5de8",
-        "sha256": "98913c88450ca7fc42b0314c2098f5ca4c455b033943cfca04fd592dcc8dd30b",
-        "md5": "c51e9ae0e46258f6ae26d22f985f1a96"
-    },
-    "4.4.0.6": {
-        "sha1": "69b3d431412e8cb6196a8112c02456e47d3e4c7e",
-        "sha256": "41fb0522fd31cd35ed2c011714e4428cdb09fa7c3455a86380cad40a437a78d1",
-        "md5": "50b517bc652b4904d6c8948e37dafbb3"
-    },
-    "4.2.0.11": {
-        "sha1": "cf0de70724c771c3edca68a5478074f193cd8d3c",
-        "sha256": "2e7a41c29905491f6af12261c41947f84bd4b76e3ccf91ea2958652f6c740f9a",
-        "md5": "335c459010f561b617bd2cc501091b1a"
-    },
-    "4.4.0.5": {
-        "sha1": "69db65a512b3e23ed212b734ac29773b7116a862",
-        "sha256": "641e8d71d6e5a173914b1dfddca2d6607f85e042b7a23316fffc04238d8204bf",
-        "md5": "bb696a61099b2983db2dc3db84f58642"
-    },
-    "4.4.0.4": {
-        "sha1": "bf87fda676d0b26d3b9d4e070c1ddb0a12282cae",
-        "sha256": "6daff63224bf9017ff358cd622c4761491fbb67ef47a72f1a81ab7f055857966",
-        "md5": "13b0a946c21d8816c4450cfc6db4948b"
-    }
+  "5.0.0-rc02": {
+    "sha1": "c52f7dc54762297b772036dab163d07375908631",
+    "sha256": "6b2969665f24bc27e18eb04c0c275a40b5075ffcb58aacd8e6a53c01a6e1fbff",
+    "md5": "94ec922d16e650e5c52d1b338f1e55df"
+  },
+  "5.0.0-rc01": {
+    "sha1": "54c27ef7d014a14ed1e10ca7b7c94487aea66bf2",
+    "sha256": "528d3b13ca539e598a1b84f80f3517151efa81d2ec7537ca932c531caf4b2ac0",
+    "md5": "e75d77aa76ca178ebf1a2efede673b90"
+  }
```

Comment:
Note these versions have been erased from the other json file, I suppose that's normal cause there is no release for neo4j